shelley/chain-and-ledger/executable-spec/test/shelley-spec-ledger-test.cabal

```diff
@@ -53,6 +54,7 @@ library
   build-depends:
     aeson,
     base,
+    base16-bytestring,
```

Comment:
Is this used?

Reply:
Turns out it's needed by the keys module